### DISCUSSION TITLE
Acesso ao require client-side sem o node

### DISCUSSION THREAD

**Discussion by JorgeBaes** [frontendbr/forum#1826] [Perguntas e Respostas]
Eu quero desenvolver um app q tem como dependencia o lowdb, para uma base de dados.
E minha API consiste em colocar dados e retirar dados e mostrá-los em um HTML. Só que o problema é que quando eu rodo o `  const { low } = require(['lowdb']) ` ele não reconhece o require como um método porque de fato não é (pq o node nao ta no cliente)

Alguém sabe como eu consigo rodar o require no script q roda no HTML

**Comment by JorgeBaes** [comment]
Bom eu achei uma solução que não era oque eu pretendia fazer. Utilizando o express e o socket.io eu criei um server que roda com node, daí eu passo os scripts do lowdb para o server-side e no client-side eu apenas faço request com o socket.io. Deu super certo.

**Comment by felquis** [comment]
Não conheço o module lowdb, o que venho fazendo em meus projetos é adaptar alguns modulos para funcionar com ES modules https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules

Funciona muito bem, porém a maioria das libs publicadas no npm não tem suporte para ES modules hoje em dia.
